core/src/main/scala/io/finch/internal/ToResponse.scala

```diff
@@ -56,6 +58,21 @@ object ToResponse extends LowPriorityToResponseInstances {
     rep
   }
 
+  implicit def asyncStreamToResponse[A, CT <: String](implicit
```

Comment:
Probably, I wasn't super clear on what I mean. My idea was to actually stream JSON array `{ obj, obj, obj }`, not a sequence of objects. Given that we now can ask for a JSON encoder, we can define this behavior pretty simple. I had something like [this in mind](https://gist.github.com/vkostyukov/463b9a7c992e2bf93fca).

I'm also thinking that we can expose that via the `ToResponse[AsyncStream[Buf]]` instance (reuse it) in some way.